db/migrate/20150616120237_create_activities_and_notifications.rb

```diff
@@ -0,0 +1,28 @@
+class CreateActivitiesAndNotifications < ActiveRecord::Migration
+  def change
```

Comment:
Assignment Branch Condition size for change is too high. [16/15]<br>Method has too many lines. [19/10]